File: Java/SauceDemo/src/main/java/pages/CartPage.java
package pages;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;

public class CartPage {

    WebDriver driver;

    By checkout_btn = By.id("checkout");
    public CartPage(WebDriver driver) {
        this.driver = driver;
    }

    public void clickCheckoutButton() {
        driver.findElement(checkout_btn).click();
    }
}

File: Python/src/pages/cart_page.py
"""
This module contains the elements of the cart page
"""

from selenium.webdriver.common.by import By
from src.pages.side_bar import Sidebar

class CartPage(Sidebar):
    """
    This class contains the elements of the cart page
    """
    def __init__(self, driver):
        super().__init__(driver)

        # Locator
        self.checkout_btn = (By.ID, "checkout")

    def click_checkout_button(self):
        self.driver.find_element(*self.checkout_btn).click()
